### PR TITLE
refactor: refactor anything server.

### PR DESCRIPTION
**Pull request by re2zero**
Export entry function and change library name back for old DFM; The backend will run with single instance.

Log: export entry function for share library.

### CODE REVIEW

File: server/backend/anythingbackend.h [hunk 1]
```diff
@@ -36,21 +36,29 @@ class AnythingBackend : public QObject
...
     int writeMountInfo();
+
+    Server *server = nullptr;
+    bool hasconnected = false;
 };
 
 DAS_END_NAMESPACE
+
+#define ANYTHINGBACKEND_SHARED_EXPORT Q_DECL_EXPORT
+extern "C" ANYTHINGBACKEND_SHARED_EXPORT int createAnything();
```
Comment (wangrong1069) [inline server/backend/anythingbackend.h:62]: fireAnything() 和 createAnything() 不一致

File: server/backend/anythingbackend.h [hunk 2]
```diff
@@ -36,21 +36,29 @@ class AnythingBackend : public QObject
...
+    static AnythingBackend *instance();
+
+    int init_connection()noexcept;
 
 protected:
 
 
 private:
-    void init_connection()noexcept;
+    int monitorStart();
```
Comment (wangrong1069) [inline server/backend/anythingbackend.h:49]: 建议将导出的函数 fireAnything() 独立放一个头文件，避免对外暴露 AnythingBackend 类

File: server/backend/anythingbackend.h (hunk 1 above)
Comment (wangrong1069) [inline server/backend/anythingbackend.h:62]: fireAnything() 和 createAnything() 不一致

**Review by wangrong1069 — COMMENTED** [review]
please check the comments

File: server/backend/anythingbackend.h (hunk 2 above)
Comment (re2zero) [reply server/backend/anythingbackend.h:49]: great idea.

**Comment by re2zero** [comment]
/review @wangrong1069 
Could you please review the latest version again, thx.

**Review by wangrong1069 — COMMENTED** [review]
looks good to me.

**Comment by wangrong1069** [comment]
/+1

**Review by wangrong1069 — APPROVED** [review]

**Pull request merged**